src/Controller/MercureController.php

```diff
@@ -47,7 +47,17 @@ public function index()
                 new Key('!ChangeMe!')
             );
 
-        $cookie = new Cookie('mercureAuthorization', $token, (new \DateTime())->add(new \DateInterval('PT24H')), '/.well-known/mercure', null, false, true, false, 'strict');
+        $cookie = new Cookie(
+            'mercureAuthorization',
+            $token,
+            (new \DateTime())->add(new \DateInterval('PT24H')),
+            '/.well-known/mercure',
+            'localhost',
```

Comment:
this was `null` before, which by HTTP Cookie specification means the current host, which is `localhost:8000`.